shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -157,12 +157,19 @@ - (void)tearDown {
   self.messageSent = nil;
 }
 
+- (FlutterViewController*)createFlutterViewController:(FlutterEngine*)engine {
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:engine
+                                                                                nibName:nil
+                                                                                 bundle:nil];
+  // The FlutterViewController's FlutterViewLayer Opaque default is NO
+  XCTAssertFalse(viewController.isViewOpaque);
```

Comment:
I don't think it is a good idea to hide this assert in a shared method that is used by all the tests.
There might be some time in the future where some headless FlutterViewController requires `isViewOpaque` to be true. It can also be overlooked because the assertion is implicitly hiding in a utility method.

It is better just to create a new test case and test this property explicitly.